docs/cudf/source/cudf_pandas/how-it-works.md

```diff
@@ -38,14 +38,19 @@ mode](api.options) is automatically enabled, ensuring consistency with
 pandas-specific semantics like default sort ordering.
 
 
-`cudf.pandas` uses a managed pool memory by default, that will also enable
+`cudf.pandas` uses a managed memory pool by default. This allows `cudf.pandas` to process datasets larger than the memory of the GPU(s) it is running on. Managed memory prefetching is also enabled by default to improve memory access performance. For more information on CUDA Unified Memory (managed memory), performance and prefetching, see [this NVIDIA Developer blog post](https://developer.nvidia.com/blog/improving-gpu-memory-oversubscription-performance/).
 prefetching <<LINK to prefetch tutorial/demo>>.
 
-There are various memory allocators that can be used by changing the environment
-variable `CUDF_PANDAS_RMM_MODE`. It supports:
+Other memory allocators can be used by changing the environment
+variable `CUDF_PANDAS_RMM_MODE` to one of the following.
 
-1. "pool"
-2. "async"
-3. "managed" (default)
-4. "managed_pool"
-5. "cuda"
+Pool allocators improve allocation performance. Without using one, memory 
+allocation may be a bottleneck depending on the workload. Managed memory
+enables oversubscribing GPU memory. This allows cudf.pandas to process 
+data larger than GPU memory in many cases, without CPU (Pandas) fallback.
```

Comment:
Maybe this should go after the list since we have "to one of the following" at the end of the paragraph above.

Reply:
Done 👍